css/styles.css

```diff
@@ -113,3 +121,90 @@ header, footer {
 .tt-cursor, .tt-suggestion:hover {
     background-color: #efefef;
 }
+
+/* improvements by YuriBrunetto */
+hgroup {
+    margin-top: 30px;
+}
+h2 {
+    font-weight: normal;
+    font-size: 16px;
+    color: #666;
+    line-height: 150%;
+}
+
+header {
+    width: 80%;
+    margin: 50px auto 0 auto;
+}
+@media all and (min-width:480px) {
+    header {
+        max-width: 450px;
+    }
+}
+
+.zen-logo {
+    width: 100px;
+    height: 100px;
+    display: block;
+    overflow: hidden;
+    border-radius: 6px;
+    margin: 0 auto;
+}
+
+.input {
+
+}
+@media all and (min-width:480px) {
+    .input-search {
+        height: 50px;
+        font-size: 18px !important;
+        color: #666 !important;
+    }
+    .btn {
+        height: 50px;
```

Comment:
This looks vertically stretched too much, if we continue with this height we'll need to increase the width and increase the icon size (possibly via `font-size`).

<img width="504" alt="capturfiles_3" src="https://cloud.githubusercontent.com/assets/1480063/13661154/ef07b44c-e644-11e5-9e6d-c958f3b5fbe8.png">